### ISSUE TITLE
Verification is not tagged when triggered from webhook with PactUri

### ISSUE THREAD

**Issue by arindamdat** [pact-foundation/pact-net#281]
I have a separate verification pipeline which gets triggered from Broker with specific Pact Uri.
This is the the code:
```
pactVerifier.PactUri(_testSettings.PactUri,
                        options: new PactUriOptions(_testSettings.PactBrokerToken),
                        providerVersionTags: new[] { _testSettings.ProviderBranch });
```
I expected that upon completion of verification, the verification will be tagged with the provider branch. But this is not happening. It seems the parameter `providerVersionTags` has no effect at all. I am using latest version.

**Comment by mefellows** [comment]
What does debug logging show of you enable it?

**Comment by bethesque** [comment]
I agree it needs to be fixed, but are you finding is actually an issue? If you're using the git sha or other determinate value (as [recommended](https://docs.pact.io/getting_started/versioning_in_the_pact_broker#best-practices)) then when the "pact changed" build runs, it will have the same provider version as when the "provider changed" (normal) pipeline was run. The tags should have been applied already in the normal pipeline.

**Comment by mefellows** [comment]
Ah, I see, this is specifically for that verification path.

**Comment by arindamdat** [comment]
Yes, I agree, the normal pipeline would have tagged it with "branch name". Still, somehow I expected this to happen from verification pipeline too since I used that parameter `providerVersionTags `. I kind of started my workshop in a weird way. I actually deleted the entire Integration from Broker. Then I tried to deploy the Consumer first (I knew it'd fail). This in turn triggered the Provider verification pipeline. Here, upon successful verification I expected the verification to be tagged with ProviderBranch. I saw this expected behavior with the Javascript library, but doesn't happen with dotnet library. That's why I reported. But it's not something blocking and if you want you can close this.
